manifest/src/main/resources/manifest.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
This file should be ignored (manifest/src is in .gitignore but I guess you added this file before updating .gitignore)

Reply:
Yes, good spot, there were a few files that got committed before I sorted the .gitignore, I'll sort it.